Makefile

```diff
@@ -4,22 +4,23 @@ TEST_DIR=tests
 
 PYTHON_VERSION ?= $(shell python --version | cut -c8- | cut -d'.' -f 1-2) # e.g. 3.9
 PY_TARGET=py$(subst .,,$(PYTHON_VERSION)) # e.g. py39
+LOG_LEVEL ?= INFO
```

Comment:
I wonder if we need `LOG_LEVEL`. Maybe the makefile could directly set  `HT_LOG_LEVEL` which is less intrusive and clearly defined.

Reply:
HT_LOG_LEVEL sounds to me more like an implementation detail. What I expect user to do is `make test LOG_LEVE=DEBUG`.